app/controllers/ChoiceController.scala

```diff
@@ -39,6 +39,7 @@ class ChoiceController @Inject()(
   errorHandler: ErrorHandler
 )(implicit ec: ExecutionContext)
     extends FrontendController with I18nSupport {
+  import forms.Choice.AllowedChoiceValues._
```

Comment:
This import can be moved to other imports section, outside the controller class.